frontend/src/components/MapView.jsx

```diff
@@ -1,31 +1,19 @@
-import * as THREE from 'three';
+import { convertToRenderCoordinates, tileConfig } from "../utils/render_utils";
 
-const tileConfig = {
-    width: 1,
-    height: 0.25,
-}
-
-// Converts map coordinates to render coordinates.
-function convertToRenderCoordinates(mapVector) {
-    return new THREE.Vector3(
-        mapVector.x * tileConfig.width,
-        mapVector.z * tileConfig.height,
-        mapVector.y * tileConfig.width
-    )
-}
-
-// Produces the geometry for a map tile.
 function renderTile(tile) {
     const position = convertToRenderCoordinates(tile)
+    const spacer = 0.00; // add space between tiles to see better, remove after improved textures
+
     return (
-        <mesh position={ [position.x, position.y, position.z] } key={`${tile.x},${tile.y},${tile.z}`}>
-            <boxGeometry args={ [tileConfig.width, tileConfig.height, tileConfig.width] } />
+                                                // place tiles below other objects
```

Comment:
Nit: I'm guessing you had extra spacing on the comment here to clarify what you're referring to? I would just be more specific in the comment to make it clear, e.g. "Reduce y-value to place tiles below other objects."

Reply:
Hmm, that's right...but I seem to have deleted the y-offset. I think we can play around a bit, but either all the enemies and turrets should have their z-position one unit higher for the tile they're on, or all tiles should be shifted down by one unit.